### ISSUE TITLE
twin anemia Polycythemia sequence

### ISSUE THREAD

**Issue by carlotarodo** [obophenotype/human-phenotype-ontology#7017]
**Preferred term label:**
twin anemia Polycythemia sequence

**Synonyms**
TAPS

**Definition (free text, please give PubMed ID)**
Chronic form of unbalanced feto-fetal transfusion through small placental anastomoses in monochorionic twin pregnancies, leading to anemia in the donor twin and polycythemia in the recipient twin.
PMID: 27068715

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0001-8801-5677

**Comment by patt1morgan** [comment]
@DominicIliescu, @sparkste.  This is another new term request that we would ask you to review and comment...thank you in advance for your participation.
